collectors/ioping.plugin/metadata.yaml

```diff
@@ -33,17 +33,38 @@ overview:
       description: ''
 setup:
   prerequisites:
-    list: []
+    list: |
+      It is necessary to have the command `ioping` installed on host. It is possible to install command passing as argument for plugin the option `install`:
```

Comment:
It is necessary to have the command `ioping` installed on your host. You can install the command by passing the argument `install` to the plugin:

Reply:
Thank you @sashwathn , I push a commit with your suggestion.